File: src/main/java/org/micoli/php/peerNavigation/configuration/PeerNavigationConfiguration.java
package org.micoli.php.peerNavigation.configuration;

import io.swagger.v3.oas.annotations.media.Schema;

public final class PeerNavigationConfiguration {
    @Schema(description = "Array of one-way navigation rules")
    public Peer[] peers = new Peer[] {};

    @Schema(description = "Array of bidirectional navigation rules")
    public Associate[] associates = new Associate[] {};
}

File: src/main/kotlin/org/micoli/php/peerNavigation/configuration/PeerNavigationConfiguration.kt
package org.micoli.php.peerNavigation.configuration

import io.swagger.v3.oas.annotations.media.Schema

class PeerNavigationConfiguration {
    @Schema(description = "Array of one-way navigation rules") var peers: Array<Peer> = arrayOf()

    @Schema(description = "Array of bidirectional navigation rules")
    var associates: Array<Associate> = arrayOf()
}
